pkg/controller/keycloak/keycloak_reconciler.go

```diff
@@ -0,0 +1,42 @@
+package keycloak
+
+import (
+	kc "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"github.com/keycloak/keycloak-operator/pkg/common"
+	"github.com/keycloak/keycloak-operator/pkg/model/keycloak"
+)
+
+type Reconciler interface {
+	Reconcile(clusterState *common.ClusterState, cr *kc.Keycloak) (common.DesiredClusterState, error)
+}
+
+type KeycloakReconciler struct {
```

Comment:
This one is definitely false-positive. We will probably also have `RHSSOReconciler` in the same file (or somewhere nearby).

Reply:
Alternatively, I can move it to `controller` (one level up). But then, we'll have one fat package with all stuff in it. I'm not sure if I like it.

Reply:
I would much prefer to leave the name as `KeycloakReconciler`, happy to leave this lint on as it would be useful in other places and possibly bypass it with a comment if that is possible with GolangCI?

Reply:
I tried to look it up but unfortunately I haven't found anything. Maybe @abstractj knows about anything like that.

Reply:
ooh wait... just found this: `// nolint`

Reply:
Works like a charm!